workflow/Snakefile

```diff
@@ -46,7 +47,15 @@ rule experiment_all:
 rule showcase_all:
     """Target rule for showcase workflow."""
     input:
-        [],
+        expand(
+            OUT_ROOT
+            / "showcases/{run_id}/{init_time}/{init_time}_{param}_{projection}_{region}.gif",
```

Comment:
should we use the same logic as with experiments using EXPERIMENT_HASH to organize outputs?

Reply:
I like it! but maybe to be implemented in a separate PR since it touches also at the experiment workflow?

Reply:
tracked in #67